optimade/server/logger.py

```diff
@@ -65,3 +72,10 @@
 
     # Add handler to LOGGER
     LOGGER.addHandler(FILE_HANDLER)
+
+try:
+    from optimade.server.config import CONFIG
+
+    LOGGER.info(f"Loaded settings from {CONFIG.config_file}.")
```

Comment:
I think this may result in several of these messages each time `LOGGER` is imported from this file/module.
Instead, which is more intuitive as well, this should probably be placed in the module invoking the server application, i.e., in `main.py` and `main_index.py`. What do you think?

Reply:
I think your suggestion is the right approach, but I don't think this would lead to multiple calls of this, otherwise it would also be setting up the logger multiple times

Reply:
Hmm, true. Still, a more "clear" logging in the server's start functionality makes more sense to me :) So happy you agree.